enhancements/network/allow-mtu-changes.md

```diff
@@ -0,0 +1,285 @@
+5. Once all the previous pods finish successfully, deploy other set of pods with
+   `restartPolicy: Never` on every node that will handle the actual change of
+   the MTU (explained in more detail below). Wait for them to be ready and
+   running.
+6. Ensure that the configmap ovnkube-config is synchronized with the new MTU
+   value.
+7. If any previous steps (1-6) was unsuccesful, the CNO will set the
+   `clusteroperator/network` conditions to:
+   - Progressing: false
+   - Degraded: true
+   Update the operator configuration status with a description of the problem.
+   At this point the process is interrupted and we require manual intervention.
+8. Force a rollout of the ovnkube-node daemonset. This will ensure
+   ovn-kubernetes uses the new MTU value for new pods as well as set the new MTU
+   on managed node interfaces like ovn-k8s-mp0, ovn-k8s-gw0 (local gateway mode)
+   and related routes.
+9. Set the new MTU value to the applied-cluster config map AND wait for pods of
+   step 3 to complete successfully.
+10. If any of the previous steps (8,9) failed, reboot the node, wait for the
+    kubelet to be reporting as Ready again.
+    If this step fails, set conditions to:
+    - Progressing: false
+    - Degraded: true
+    Update the operator configuration status with a description of the problem.
+11. Upon completion, set conditions to:
+    - Progressing: false
+    - Degraded: false
+
+The steps to change the MTU performed by pods of previous step 3 are:
+1. So that we don't have nodes doing things at different times and we have
+   everything synchronized, the pods will wait until the MTU value on the
+   applied-cluster configmap changes.
+2. Enter every network namespace. If an interface `eth0` exists in that
+   namespace with an ip address within the pod subnet, change the MTU of the
+   the veth pair.
+3. If any of these steps failed (1-3), the pod will exit with code 1, if all
+   were successful it will exit with code 0.
+
+An administrator should be able to deploy a machine-config object to change
+the node MTU as well. If increasing the MTU, it will do so at the beginning
+of the procedure. If decreasing the MTU, it will do so at the end of the
+procedure.
+
+### User Stories
+
+#### As an administrator, I want to change the node MTU
+
+An administrator should be able to deploy a machine-object config object
+that configures the node MTU permanently. Ideally this would be achieved
+through the ability to run configure-ovs with an MTU parameter.
+configure-ovs should change the MTU of br-ex and ovs-if-phys0 with the
+least impact on the existing configuration to avoid any unnecessary
+disruption. This change should persist across reboots.
+
+#### As an administrator, I want to change the cluster network MTU
+
+An administrator should be able to change the cluster network MTU through
+CNO configuration change. This would encompass the following tasks:
+
+##### Implement a pod that changes the actual MTU on running pods
```

Comment:
So again, the parts talking about implementation details don't belong in "User Stories". And they're redundant with what you've already said, so you can just remove them.

Reply:
I will move them to a specific section. The thing is that there is no way to map (user) stories here to (non-user) stories in Jira.